twice_decorator.py

```diff
@@ -0,0 +1,47 @@
+from pyclbr import Function
+import decorator
+
+
+class CustomError(Exception):
+    pass
+
+
+def twice_decorator(func: Function):
+    """
+    A decorator that executes the function it wraps twice.
+
+    :param func: The function that have the property @twice_decorator that been called.
+    :return: Decorator of the inner function.
+    """
+    def wrapped_function(function, argument) -> int:
```

Comment:
Use decorator.decorator as a decorator, not as a function, to reduce boilerplate code

Reply:
I don't understand this remark - I used decorator.decorator...

Reply:
Look how it is used in the official docs - you dont use it right. 
https://github.com/micheles/decorator/blob/master/docs/documentation.md
Also, look the the implementation of decorator.decorator - especially the second argument you are giving it